docs/guides/_toc.json

```diff
@@ -246,7 +246,7 @@
               ]
             },
             {
-              "title": "Qiskit Transpiler Service",
+              "title": "Qiskit IBM Transpiler",
```

Comment:
Why this change? The product is Qiskit Transpiler Service

Reply:
this is one of the reasons why I included @jyu00 to review this PR. To me it makes no sense to keep the section as `Qiskit Transpiler Service` since now the AI passes can be run on local. And that specific feature is for the package `qiskit-ibm-transpiler`, so that's why the change

Open for suggestions, I was reluctant to change this but was the best way I found

Reply:
Should it be `Qiskit Transpiler`? For Runtime, we don't mention IBM in the name—only the package. It's called in docs `Qiskit Runtime client` for example.

The product name for `qiskit-ibm-transpiler` has thus far been called `Qiskit Transpiler Service client`.

--

We could also make this change in a follow up? Maybe revert the change for now, then settle on the name in a follow up when things are calmer?

Reply:
it's OK to me. Let's keep as is and we'll handle it in a future PR

Fixed in https://github.com/Qiskit/documentation/pull/2273/commits/8984011c935f6763b3b40b2bf6eac42d003c1b78